README.md

```diff
@@ -67,3 +67,63 @@ void loop() {
   delay(1000);
 }
 ```
+
+### Usage with two SHT31 sensors
+
+```c++
+#include <Wire.h>
+#include "SHTSensor.h"
+
+// Sensor with normal i2c address
+// Sensor 1 with address pin pulled to GND
+SHTSensor sht1(SHTSensor::SHT3X);
+
+// Sensor with alternative i2c address
+// Sensor 2 with address pin pulled to Vdd
+SHTSensor sht2(SHTSensor::SHT3X_ALT);
+
+void setup() {
+  // put your setup code here, to run once:
+  Wire.begin();
+  Serial.begin(9600);
+  delay(1000); // let serial console settle
+  // init will always be successful
+  // case Auto_detect will not occure, because mSensorType is given
```

Comment:
There are a few typos, but could you describe the problem from the user perspective, i.e. that doesn't require to know the internals of the driver (`mSensorType` is purely internal). I'd describe it as

> `init` on a specific sensor type (i.e. without auto detecting), does not check if the sensor is responding and will thus always succeed.

Feel free to come up with alternative formulations